deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/IncompatibleKerasConfigurationException.java

```diff
@@ -0,0 +1,12 @@
+package org.deeplearning4j.nn.modelimport.keras;
+
+
+public class IncompatibleKerasConfigurationException extends RuntimeException {
```

Comment:
Javadoc: People will hit this exception, and the javadoc should help answer the questions "why?" and "what should I do now?"
Maybe mention that it's WIP and to visit the dl4j issues page or gitter?